test_demo.py

```diff
@@ -8,10 +8,10 @@ def test_show_aggie_pride(self):
         df = show_aggie_pride()
         self.assertEqual(df.loc[0, 'Text'], 'Aggie Pride')
         self.assertEqual(df.loc[1, 'Text'], 'Worldwide')
-        self.assertEqual(df.loc[2, 'Text'], 'Hard Work Yields Results')        
-        self.assertEqual(df.loc[3, 'Text'], 'Work today for what you want to achieve tomorrow')
+        self.assertEqual(df.loc[2, 'Text'], 'Hard Work Yields Results')
+	self.assertEqual(df.loc[3, 'Text'], 'Work today for what you want to achieve tomorrow')
         self.assertEqual(df.loc[4, 'Text'], 'Aggies DO!')
```

Comment:
Your slogan is 'Aggies Do!' but the unit-test is still checking for 'Aggies DO!' - that should fail because of the uppercase O, so, you should change that to match your slogan string.